packages/runtime/container-runtime/src/summary/runningSummarizer.ts

```diff
@@ -837,7 +837,7 @@ export class RunningSummarizer implements IDisposable {
 		// Set to undefined first, so that subsequent enqueue attempt while summarize will occur later.
 		this.enqueuedSummary = undefined;
 		this.trySummarizeOnce(
-			{ reason: `enqueuedSummary/${reason}` },
+			{ summarizeReason: `enqueuedSummary/${reason}` },
```

Comment:
Actually, this should just be `reason` because when `this.enqueueSummary` is set, the reason is already `enqueue:${reason}`.
This adds another prefix. Fixed this and added validation in test.